### PR TITLE
test: cutler --help

### PR DESCRIPTION
**Pull request by eseidel**
Add a very basic test of `cutler --help`

The main point of this was to test the construction of Cutler.
Because Command prints usage via print, I had to capture print
with runZoned.

I'm sure we can write fancier tests here in the future if needed.

### CODE REVIEW

**Comment by eseidel** [comment]
Because the print comes from package args CommandRunner, not us.

**Comment by eseidel** [comment]
I guess we could override printUsage on CommandRunner to have it use a
logger. That’s probably better.

**Review by felangel — APPROVED** [review]
left a comment but LGTM either way ✅

File: cutler/test/cutler_test.dart [hunk 1]
```diff
@@ -80,4 +83,31 @@ void main() {
     );
     expect(fromWorkingDir, checkoutRoot.path);
   });
+
+  test('smoke test', () async {
+    // TODO(eseidel): A valid checkout root should not be needed to see --help.
+    final logger = _MockLogger();
```
[the remark this answers is not in the document]
Comment (eseidel) [reply cutler/test/cutler_test.dart:88]: Fixed.

**Pull request merged**